templates/kernel/lifecycle.html

```diff
@@ -87,7 +67,18 @@ <h2>Installation</h2>
           <p>Server:</p>
           <pre><code>sudo apt-get install --install-recommends linux-generic-hwe-18.04</code></pre>
           <p>The 18.04.2 and newer point releases will ship with an updated kernel and X stack by default for the desktop. Server installations will default to the GA kernel and provide the enablement kernel as optional.</p>
-          <p>The 18.04 HWE Stacks will follow the Rolling Update Model.</p>
+          <p>The 18.04 HWE stacks follow <a href="https://wiki.ubuntu.com/Kernel/RollingLTSEnablementStack">the Rolling Update Model</a>.</p>
```

Comment:
The copy doc points here https://ubuntu.com/about/release-cycle#ubuntu-kernel-release-cycle